File: Zadania/2025/Zadanie-7/Python/Terrorysta.py
from Gracz import Gracz
class Terrorysta:
    Gracz = Gracz()
    def __init__(self , actualWeapon):
        super().__init__(actualWeapon)
        self.czyMaBombe = False

    def podlozBombe(self):
        self.czyMaBombe = True
        print("Podklada bobmbe!")

    def changeWeapon(self , newWeapon):
        self.actualWeapon = newWeapon


        
        

File: Zadania/2025/Zadanie-7/Java/Terrorysta.java
package Java;

class Terrorysta extends Gracz{
    boolean czyMaBombe;
    public Terrorysta(String nickname ,Bron actualWeapon , boolean czyMaBombe){
        super(nickname, actualWeapon);
        this.czyMaBombe = czyMaBombe;
    }
    public void podlozBombe(){
        if (czyMaBombe){
            System.out.println(nickname + " podkłada bobmbe!");
        }else{
            System.out.println(nickname + " nie ma bomby!");
        }
    }
}
